frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -122,6 +126,24 @@ export const WorkspacePage = (): JSX.Element => {
     [isUserLoading, hasSeenAnnouncement],
   )
 
+  const EMERGENCY_CONTACT_KEY = useMemo(
```

Comment:
nit: since this is an inlined constant variable, should this be in camelCase instead?

should we also return `null` if user._id does not exist yet? so we don't have cases of `has-emergency-contactundefined` being set as a key.